include/deal.II/fe/fe_values.h

```diff
@@ -2912,6 +2914,25 @@ class FEValuesBase : public Subscriptor
   /// @name Geometry of the cell
   //@{
 
+  /**
+   * Return an object that can be thought of as an array containing all
+   * indices from zero to `n_quadrature_points`. This allows to write code
+   * using range-based for loops of the following kind:
+   * @code
+   *   FEValues<dim> fe_values (...);
+   *
+   *   for (auto &cell : dof_handler.active_cell_iterators())
+   *     fe_values.reinit(cell);
+   *     for (auto q_point : fe_values.quadrature_point_indices())
```

Comment:
too much python writing -- you need curly braces

Reply:
Drats. I snuck this in before sitting down to dinner. Syntax goes out the window when you're hungry!